### PR TITLE
[IMP] POS: add info for IOS devices

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
task-3388561

This PR takes content to generate and export an SSL certificate to add it under the "Mac" tab and adds info from the task under a new tab "IOS" to import an existing SSL certificate into an IOS device.

(There's a video of the complete flow available on the task)

### CODE REVIEW

File: content/applications/sales/point_of_sale/configuration/epos_ssc.rst [hunk 1]
```diff
@@ -197,6 +267,32 @@ it into your browser.
...
+      the configuration profile, and close the second pop-up window. Then,
+
+      #. go to the **Settings App** on the iOS device;
+      #. click :guilabel:`Profile Downloaded` under the user's details box;
+      #. locate the downloaded `.crt` file and select it;
+      #. click :guilabel:`Install` on the top right of the screen;
+      #. if a passcode is set on the device, enter the passcode;
+      #. click :guilabel:`Install` on the top right of the certificate warning screen and the pop-up
+         window;
+      #. click :guilabel:`Done`.
```
Comment (Donapi) [inline content/applications/sales/point_of_sale/configuration/epos_ssc.rst:286]: I would have capitalised the beginning of each sentence

File: content/applications/sales/point_of_sale/configuration/epos_ssc.rst [hunk 2]
```diff
@@ -197,6 +267,32 @@ it into your browser.
...
+
+      .. image:: epos_ssc/ssl-ios-verified.png
+
+      The certificate is installed, but it still needs to be authenticated. To do so,
+
+      #. go to :menuselection:`Settings --> General --> About > Certificate Trust Settings`;
+      #. enable the installed certificate using the **slide button**;
+      #. click :guilabel:`Continue` on the pop-up window.
```
Comment (Donapi) [inline content/applications/sales/point_of_sale/configuration/epos_ssc.rst:294]: I would have capitalised the beginning of each sentence

**Review by Donapi — APPROVED** [review]
Just a suggestion to capitalize :)

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**